webview/src/experiments/components/table/Table.tsx

```diff
@@ -139,29 +147,37 @@ export const Table: React.FC<TableProps & WithChanges> = ({
   useClickOutside(tableRef, clickOutsideHandler)
 
   const batchRowSelection = React.useCallback(
```

Comment:
[Q] Can we further test this?

Reply:
Sure, I think I can add some more tests today